### PR TITLE
Update project_structure.md

### PR DESCRIPTION
**Pull request by kuraga**
For better understanding of the whole structure.

### CODE REVIEW

File: docs/project_structure.md [hunk 1]
```diff
@@ -40,18 +40,27 @@ The `nginx` directory contains a NGINX configuration file and Docker container.
...
 
+Also, there are directories and files contain doccano v0.x codes.
+In the future, they will be integrated into the currect code or removed:
+
+```
+/
+├── app/
+├── └── server/
+└── docker-compose.prod.yml
+```
```
Comment (kuraga) [inline docs/project_structure.md:51]: Which files could we add here?

Comment (Hironsan) [reply docs/project_structure.md:51]: Files under `app` directory is related to doccano v0.x and v1.x. In particular, `server` directory is related to v0.x.

```
.
├── api/
├── app/
├── authentification/
├── server/
├── tools/
├── Dockerfile
├── manage.py
└── requirements.txt
```

Comment (kuraga) [reply docs/project_structure.md:51]: This paragraph is about v0.x only, look, please.

* /app/server/
* /docker-compose.prod.yml

I missed something?

Comment (Hironsan) [reply docs/project_structure.md:51]: `/docker-compose.prod.yml` is for v1.x. Change it to `/docker-compose.yml`.

Comment (kuraga) [reply docs/project_structure.md]: Ooough..... I'm sorry. Thanks! Done.

**Comment by kuraga** [comment]
Wow! So cool :smile:

File: docs/project_structure.md (hunk 1 above)
Comment (kuraga) [reply docs/project_structure.md]: @Hironsan , BTW I didn't know `app/server` is used in v1.x _default_ (Docker Compose) installation, too. But now [I see](https://github.com/doccano/doccano/blob/master/app/server/management/commands/create_admin.py).

Should I change something in this paragraph again, what do you think?

**Pull request merged**